text/0232-maturity-of-otel.md

```diff
@@ -0,0 +1,72 @@
+
+#### Beta
+
+Same as Alpha, but the interfaces (API, configuration, generated telemetry) are treated as stable whenever possible. While there might be breaking changes between releases, component owners should try to minimize them. A component at this stage is expected to have had exposure to non-critical production workloads already during its Alpha phase, making it suitable for broader usage.
+
+#### Release Candidate
+
+The component is feature-complete and ready for broader usage. The component is ready to be declared stable, it might just need to be tested in more production environments before that can happen. Bugs and performance problems are expected to be reported, and there's an expectation that the component owners will work on them. Breaking changes, including configuration options and the component's output, are only allowed under special circumstances. Whenever possible, users should be given prior notice of the breaking changes.
+
+#### Stable
+
+The component is ready for general availability. Bugs and performance problems should be reported, and there's an expectation that the component owners will work on them. Breaking changes, including configuration options and the component's output, are only allowed under special circumstances. Whenever possible, users should be given prior notice of the breaking changes.
+
+#### Deprecated
+
+Development of this component is halted, and no further support will be provided. No new versions are planned, and the component might be removed from its included distributions. Note that new issues will likely not be worked on except for critical security issues. Components that are included in distributions are expected to exist for at least two minor releases or six months, whichever happens later. They also MUST communicate in which version they will be removed, either in terms of a concrete version number or the date of a release, like: "the first release after 2023-08-01".
+
+#### Unmaintained
+
+A component identified as unmaintained does not have an active code owner. Such components may have never been assigned a code owner, or a previously active code owner has not responded to requests for feedback within 6 weeks of being contacted. Issues and pull requests for unmaintained components SHOULD be labeled as such. After 6 months of being unmaintained, these components MAY be deprecated. Unmaintained components are actively seeking contributors to become code owners.
```

Comment:
This feels out of place with everything else. In order to reach any status, it must have had a code owner at some point. If the code owner is not responding, then the component is removed if experimental or marked as deprecated if the component is already stable. This means the existing statuses cover this special circumstance. Instead of this being a maturity level it should be called out separately.

Reply:
I believe this contradicts the point above on "not implemented" if only the collector has this state 😃 

I see the value of this status, especially for developers, but for end-users consuming the release, it seems irrelevant. Even if it's unmaintained it's supported until it's deprecated.

Reply:
End-users seeing a component they rely on being "unmaintained" might start looking for alternatives or might step in to become code owners themselves (this has happened as well). I agree that developers and other community members are more interested in this than end-users.

Reply:
I'm marking this as resolved but feel free to unresolve, or to open a new issue to address this in a follow-up PR.